### PR TITLE
Translate crosswalk app tool into chinese and put it's link on the le…

### PR DESCRIPTION
**Pull request by chuan9**
…ft navigation bar

### CODE REVIEW

**Comment by chuan9** [comment]
@jondong @lisa0314 Please have a review. It seems that many people want to see it urgently

File: public/documentation/crosswalk-app-tools_zh.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
+# Crosswalk App Tools
+
+Crosswalk-app-tools用来创建和编译Crosswalk应用，是一款新的、基于NPM发布的工具。它取代了基于Python的make_apk.py脚本。由于它基于Node.js，因此是一款跨平台的工具。目前主持的平台有Windows，Linux，和Apple OS X。
```
Comment (jondong) [inline public/documentation/crosswalk-app-tools_zh.md]: 支持的平台

**Comment by jondong** [comment]
@chuan9 only a typo here, thanks for the PR.

**Comment by chuan9** [comment]
@jondong Done! Thanks for review

**Comment by jondong** [comment]
LGTM, thanks! @baleboy would you please take a look and help to merge? thanks

**Pull request merged**